codespell_lib/_codespell.py

```diff
@@ -28,7 +28,8 @@
 import sys
 import textwrap
 
-word_regex_def = u"[\\w\\-'’`]+"
+# NOTE: flake8 suppression due to it not liking \] escape sequence
+word_regex_def = u"([\\w\\-'’`]+)([.,?!-:;><@#$%^&*()_+=/\]\\[])?"  # noqa W605
```

Comment:
I'm not sure I got all of the punctuation marks here.  Oddly, flake8 doesn't really like `\]`